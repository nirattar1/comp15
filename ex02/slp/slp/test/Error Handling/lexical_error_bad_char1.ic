// And of course, there's always the usual suspects
class Cl{
	int iSpyWithMyLittleI;

	// Something beginning with:

	string $;
}

