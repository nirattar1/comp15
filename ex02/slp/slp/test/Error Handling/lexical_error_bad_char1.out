7: Lexical error: illegal character '$'